### DISCUSSION TITLE
Can someone please suggest what is the most stable version?

### DISCUSSION THREAD

**Discussion by guymoy1** [tinacms/tinacms#2437] [Help] [answered]
I having some issues with rendering my pages in a preview mode after updating to 0.62.0
Thanks.

**Comment by perkinsjr** [comment, marked as answer]
Hi @guymoy1 the latest versions are below:

```
tinacms: 0.63.0
@tinacms/cli : 0.60.0
```

if you continue to run into issues when you have updated to the latest, please provide the following

- Errors when attempting to render pages in preview mode.
- A repository so we can look at the whole picture (if possible)

**Reply by guymoy1** [reply]
Thanks,
What about react version and nextJS version that suits to the tinaCMS version?

**Reply by perkinsjr** [reply]
Next 12 / React 17 (which is the latest if you create a new project). 

It should work with previous versions of Next (10, 11) without any issues, which includes React 16.
